contract/views/contract_portal_templates.xml

```diff
@@ -165,6 +165,28 @@
                                 </tbody>
                             </table>
                         </div>
+                        <div t-if="contract.modification_ids" id="item_modifications">
+                            <table class="table table-sm" id="sales_order_table">
```

Comment:
Can this be shown as timeline?

Reply:
Please, copy some little example about it.